<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-22T16:51:00 GMT+07:00" name="Default suite" finished-at="2024-12-22T16:52:10 GMT+07:00" duration-ms="69499">
    <groups>
    </groups>
    <test started-at="2024-12-22T16:51:00 GMT+07:00" name="Default test" finished-at="2024-12-22T16:52:10 GMT+07:00" duration-ms="69499">
      <class name="automation.testsuite.qlklv.ViewAndUpdateKVLV">
        <test-method is-config="true" signature="openFireFox()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:50:51 GMT+07:00" name="openFireFox" finished-at="2024-12-22T16:51:00 GMT+07:00" duration-ms="9587" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openFireFox -->
        <test-method is-config="true" signature="Login()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:00 GMT+07:00" name="Login" finished-at="2024-12-22T16:51:03 GMT+07:00" duration-ms="2266" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method is-config="true" signature="openquanlykvlv()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:03 GMT+07:00" name="openquanlykvlv" finished-at="2024-12-22T16:51:04 GMT+07:00" duration-ms="1688" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openquanlykvlv -->
        <test-method signature="XemvaupdateKVLV001()[pri:1, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:04 GMT+07:00" name="XemvaupdateKVLV001" finished-at="2024-12-22T16:51:05 GMT+07:00" duration-ms="669" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at automation.page.QuanlyKLVPage.CapnhatKVLV(QuanlyKLVPage.java:60)
at automation.testsuite.qlklv.ViewAndUpdateKVLV.XemvaupdateKVLV001(ViewAndUpdateKVLV.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- XemvaupdateKVLV001 -->
        <test-method is-config="true" signature="openquanlykvlv()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:05 GMT+07:00" name="openquanlykvlv" finished-at="2024-12-22T16:51:07 GMT+07:00" duration-ms="1489" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openquanlykvlv -->
        <test-method signature="XemKVLV002()[pri:2, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:07 GMT+07:00" name="XemKVLV002" finished-at="2024-12-22T16:51:15 GMT+07:00" duration-ms="8170" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- XemKVLV002 -->
        <test-method is-config="true" signature="openquanlykvlv()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:15 GMT+07:00" name="openquanlykvlv" finished-at="2024-12-22T16:51:16 GMT+07:00" duration-ms="1477" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openquanlykvlv -->
        <test-method signature="XemKVLV003()[pri:3, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:51:16 GMT+07:00" name="XemKVLV003" finished-at="2024-12-22T16:52:08 GMT+07:00" duration-ms="51392" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at automation.testsuite.qlklv.ViewAndUpdateKVLV.XemKVLV003(ViewAndUpdateKVLV.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- XemKVLV003 -->
        <test-method is-config="true" signature="openquanlykvlv()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:52:08 GMT+07:00" name="openquanlykvlv" finished-at="2024-12-22T16:52:09 GMT+07:00" duration-ms="1677" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openquanlykvlv -->
        <test-method signature="SuaKVLV001()[pri:4, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:52:09 GMT+07:00" name="SuaKVLV001" finished-at="2024-12-22T16:52:10 GMT+07:00" duration-ms="569" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at automation.page.QuanlyKLVPage.CapnhatKVLV(QuanlyKLVPage.java:60)
at automation.testsuite.qlklv.ViewAndUpdateKVLV.SuaKVLV001(ViewAndUpdateKVLV.java:69)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuaKVLV001 -->
        <test-method is-config="true" signature="closeWebPage()[pri:0, instance:automation.testsuite.qlklv.ViewAndUpdateKVLV@2cf3d63b]" started-at="2024-12-22T16:52:10 GMT+07:00" name="closeWebPage" finished-at="2024-12-22T16:52:12 GMT+07:00" duration-ms="1520" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeWebPage -->
      </class> <!-- automation.testsuite.qlklv.ViewAndUpdateKVLV -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
